### ISSUE TITLE
[Accessibility][Product Carousel] The "previous" and "next" buttons in the carousel have no visible indication of focus

### ISSUE THREAD

**Issue by LevelAccessA11y** [adobe/aem-core-wcm-components#1764]
**Steps to Reproduce**
1. Open page in Google Chrome.
2. Attempt to navigate to the "previous" and "next" buttons by using TAB or TAB + SHIFT keys.

**Current Behavior**
The "previous" and "next" buttons provide no visual indication when a user is focused on either of them. Visual keyboard users cannot determine which button has focus.

**Expected behavior/code**
When actionable elements are focused, ensure that visual indication is provided. Visual focus is most often provided via an outline and it needs to meet a 3:1 color contrast ratio with the background.

**Environment**
Chrome 91, Windows 10
AEM Core Components v. 2.17  

**Affected Users**
Users with low vision

**WCAG SC**
2.4.7 Focus Visible

**Comment by richardhand** [comment]
@adobe export issue to Jira project SITES

**Comment by mhaack** [comment]
Fixed by CIF-2279 via https://github.com/adobe/aem-core-cif-components/pull/666
